chef/cookbooks/provisioner/files/default/crowbar_join.init.suse

```diff
@@ -4,7 +4,7 @@
 # Required-Start:    $syslog $network $remote_fs sshd
 # Should-Start:      neutron-ovs-cleanup xend libvirtd
 # Required-Stop:     $syslog $network $remote_fs sshd
-# Should-Stop:       openvswitch-switch ovs-usurp-config-br-fixed ovs-usurp-config-br-public
+# Should-Stop:       ovs-usurp-config-br-fixed ovs-usurp-config-br-public
```

Comment:
rpm -qpl shows
openvswitch.service
ovs-vswitchd.service
ovsdb-server.service

so maybe we should add ovs-vswitchd too?